packages/css/src/components/row/row.scss

```diff
@@ -19,3 +19,47 @@
 .ams-row--wrap {
   flex-wrap: wrap;
 }
+
+.ams-row--align-around {
+  justify-content: space-around;
+}
+
+.ams-row--align-between {
+  justify-content: space-between;
+}
+
+.ams-row--align-center {
+  justify-content: center;
+}
+
+.ams-row--align-end {
+  justify-content: flex-end;
+}
+
+.ams-row--align-evenly {
+  justify-content: space-evenly;
+}
+
+.ams-row--align-start {
+  justify-content: flex-start;
+}
+
+.ams-row--align-vertical-bottom {
```

Comment:
Don't we want to use `end` here, instead of `bottom`? We're already asking are users to familiarize themselves with `start` and `end`, might as well do it consistently, no?

Reply:
I think either approach would be justifiable.

I aim for names that are easy to guess and close to common mental models. My main goal for a design system is helping people from different disciplines work together on digital products. One dream is that we’ll have some kind of environment where people with all kinds of roles, not just developers, design in the browser, writing our API.

In that light, I would say reserving ‘start’ and ‘end’ for horizontal and using ‘top’ and ‘bottom’ for vertical is a bit more straightforward. I guess people need to process ‘start’ meaning the top of a column a little bit longer than ‘top’.

Reply:
We’ll go for ‘start’ and ‘end’ in both directions.

Reply:
Updated.